lib/mongoid/scroll/base64_encoded_cursor.rb

```diff
@@ -0,0 +1,43 @@
+require 'base64'
+require 'json'
+
+module Mongoid
+  module Scroll
+    # Allows to serializer/deserialize the cursor using RFC 4648
+    class Base64EncodedCursor < Cursor
+      class << self
+        def from_cursor(cursor)
+          base64_encoded_cursor = new(nil,
+                                      field_type: cursor.field_type,
+                                      field_name: cursor.field_name,
+                                      direction: cursor.direction,
+                                      include_current: cursor.include_current)
+          base64_encoded_cursor.value = cursor.value
+          base64_encoded_cursor.tiebreak_id = cursor.tiebreak_id
+          base64_encoded_cursor
+        end
+      end
+
+      def to_s
+        Base64.strict_encode64({ value: super, field_type: field_type, field_name: field_name, direction: direction, include_current: include_current }.to_json)
```

Comment:
Nit: it's a long line, maybe put each value on its own - it's also easier to read diffs.

```
({
  value: super,
  field_type: ...
   ...
})
```